x/pairing/keeper/msg_server_relay_payment.go

```diff
@@ -68,6 +84,14 @@ func (k msgServer) RelayPayment(goCtx context.Context, msg *types.MsgRelayPaymen
 			)
 		}
 
+		_, found := k.specKeeper.GetSpec(ctx, relay.SpecId)
```

Comment:
remove, this is not needed, happens in relay payment

Reply:
done. see https://github.com/lavanet/lava/commit/b923474cd40ba656fc06008c7eb36a84a6652409